### DISCUSSION TITLE
Multiple Selections

### DISCUSSION THREAD

**Discussion by peteswan** [joshhanley/livewire-autocomplete#22] [Ideas]
Hi mate,

This is great thank you.

I want to use it as a tagging interface. Works great for one autocomplete, but would like to be able to do multiple, eg type, click the tab or add new, then type again and do the same.

Does this package do this or can it be implemented?

Cheers

**Comment by joshhanley** [comment]
@peteswan thanks! Currently, it doesn't support this, but it's something I'm looking at adding in 🙂
